src/plugins/data/public/search/search_service.ts

```diff
@@ -53,9 +53,7 @@ export class SearchService implements Plugin<ISearchSetup, ISearchStart> {
 
   private getSearchStrategy = <T extends TStrategyTypes>(name: T): TSearchStrategyProvider<T> => {
     const strategyProvider = this.searchStrategies[name];
-    if (!strategyProvider) {
-      throw new Error(`Strategy with name ${name} does not exist`);
-    }
+    if (!strategyProvider) throw new Error('Search strategy not found');
```

Comment:
Nit: I think it's somewhat useful to have the name of the missing search strategy as part of the error message, but I'm honestly fine either way